enderio-base/src/main/java/com/enderio/base/common/item/darksteel/upgrades/EmpoweredUpgrade.java

```diff
@@ -158,18 +63,27 @@ public Component getDisplayName() {
     }
 
     @Override
-    public Tag serializeNBT() {
-        CompoundTag nbt = new CompoundTag();
-        nbt.putInt(TIER_KEY, tier.ordinal());
+    protected EmpoweredUpgradeTier getBaseTier() { return EmpoweredUpgradeTier.ONE; }
```

Comment:
seperate method to multiple lines
```suggestion
    protected EmpoweredUpgradeTier getBaseTier() {
        return EmpoweredUpgradeTier.ONE;
    }
```